### PR TITLE
Geth

### PR DESCRIPTION
**Pull request by govale**
*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

**Comment by unguiculus** [comment]
See my comments in #3643.

File: incubator/geth/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: "arilot/geth"
-  tag: "alpine"
+  tag: "latest"
```
Comment (unguiculus) [inline incubator/geth/values.yaml]: Use a stable image

File: incubator/geth/values.yaml [hunk 2]
```diff
@@ -28,7 +29,7 @@ persistence:
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
-resources: {}
-#  requests:
-#    memory: 512Mi
-#    cpu: 300m
+resources:
```
Comment (unguiculus) [inline incubator/geth/values.yaml]: Don't specify defaults for resources.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/geth/values.yaml (hunk 1 above)
Comment (daniel-yavorovich) [reply incubator/geth/values.yaml]: fixed. also I have change image to official

File: incubator/geth/values.yaml (hunk 2 above)
Comment (daniel-yavorovich) [reply incubator/geth/values.yaml]: fixed

**Comment by daniel-yavorovich** [comment]
@unguiculus please review changes

File: incubator/geth/values.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+image:
+  repository: "ethereum/client-go"
+  tag: "stable"
```
Comment (unguiculus) [inline incubator/geth/values.yaml]: Please use a release tag.

**Review by unguiculus — CHANGES_REQUESTED** [review]
You need to merge master to make CircleCI happy. I'd suggest you move the chart to stable before we merge it. And have a look at the CLA issue.

**Comment by daniel-yavorovich** [comment]
@unguiculus we have merged master, but we cant fix cla issue. Do you have any info about reason of failed check?

govale (author of PR) have not maintained this chart already. May be I need recreate pull request from my account?

Thank you.

**Pull request closed without merging**